client/packages/common/src/hooks/useSimplifiedTabletUI/useSimplifiedTabletUI.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * This hook determines whether or not the UI should be displayed with the
+ * "simplified" options when on a mobile device.
+ *
+ * Use the value returned from this hook whenever a UI element is conditional on
+ * the "simplified" mobile view.
+ *
+ * Three criteria must be true:
+ * - The store pref "Use Simplified Mobile UI" is enabled
+ * - The (old) preference "Pack size to 1" is enabled
+ * - The device screen size is "tablet" size or smaller
+ */
```

Comment:
It was suggested in [this comment](https://github.com/msupply-foundation/open-msupply/issues/7388#issuecomment-2889305660) that the "pack to one" option maybe should only be considered in places where it's relevant. For now, this is checking it for *all* situations.

If we want to change this, I can easily add this as an argument for this hook so we can change it for different cases. What do we think?

Reply:
I think we leave it how you have it for now. Makes sense to me.
Should really be a warning on adding a Simplified UI saying it won't work properly without Packsize to 1...